activation.cpp

```diff
@@ -89,6 +91,41 @@ void Activation::unsubscribeFromSystemdSignals()
     return;
 }
 
+std::string Activation::getBuildID(const std::string& manifestPath)
```

Comment:
We can probably get rid of this function. We have the "getValue()" function from version.cpp that can be called to get any field from the manifest file, so we can do the regex comparison in the activation function before calling to verify the UAK instead of calling this getBuildID function.